zbench.zig

```diff
@@ -152,6 +152,43 @@ pub const Results = struct {
         for (self.results) |r| try r.prettyPrint(writer, colors);
     }
 
+    /// Prints a summary output at the end of benchmarking sessions
+    /// This summary highlights the fastest benchmark by name in green, and compares each
+    /// subsequent benchmark to show how many times slower they are relative to the fastest.
+    pub fn printSummary(self: Results, writer: anytype) !void {
+        if (self.results.len == 0) return;
+
+        const bold = Color.bold();
+        const resetCode = Color.reset.code();
+        const greenCode = Color.green.code();
+        const blueCode = Color.blue.code();
+
+        // Find the fastest result without sorting
+        var fastestIndex: usize = 0;
+        var fastestTime = self.results[0].statistics.mean_ns;
+        for (self.results, 0..) |res, i| {
+            if (res.statistics.mean_ns < fastestTime) {
+                fastestTime = res.statistics.mean_ns;
+                fastestIndex = i;
+            }
+        }
+
+        // Print Summary Heading in Bold
+        try writer.print("\n \n{s}Summary{s}\n", .{ bold, resetCode });
+
+        try writer.print("{s}", .{greenCode});
+        try writer.print("{s}", .{self.results[fastestIndex].name});
+        try writer.print("{s} ran\n", .{resetCode});
+
+        for (self.results, 0..) |res, i| {
+            if (i == fastestIndex) continue; // Skip the fastest since it's already printed
+            const times = @as(f64, @floatFromInt(res.statistics.mean_ns)) / @as(f64, @floatFromInt(fastestTime));
```

Comment:
A small improvement might be to define `inv_fastest_time = 1 / @as(f64, @floatFromInt(res.statistics.mean_ns));` up above, then you don't need to do the cast and divide, just a multiply. Btw, the general naming convention in Zig seems to be snake case for variables and camel case for functions: https://ziglang.org/documentation/master/#Names.